### ISSUE TITLE
Cache options

### ISSUE THREAD

**Issue by Planeshifter** [kgryte/node-module-starter-kit#1]
When using the CLI app to create modules, it would be nice if there was functionality to specify default options used when the command is invoked without setting them. For example, one might want to always use the same values for the author and email options. To facilitate this, one could provide the user with a command to store values in a `config.json` file in his or her home directory, from which the default values would then be read.

**Comment by kgryte** [comment]
Yeah, I have that in the `TODO`, possibly using `configstore`, or storing in the user's config directory.
